### PR TITLE
Issue #32 remove member word

### PR DESCRIPTION
**Pull request by Gianni-Pucillo**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
 			"integrity": "sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==",
```
Comment (shciubrcom) [inline package-lock.json]: `package-lock.json` shouldn't be updated by this issue.

**Review by shciubrcom — CHANGES_REQUESTED** [review]
`package-lock.json` shouldn't be updated by this issue.

File: package-lock.json (hunk 1 above)
Comment (Gianni-Pucillo) [reply package-lock.json]: Please, how can I avoid this should happens again?
Can you solve this problem or have I to perform any step?

Thanks.

Comment (shciubrcom) [reply package-lock.json]: You can use `npm ci` instead `npm install` to avoid modifications in package-lock.json

Comment (Gianni-Pucillo) [reply package-lock.json]: Ok got it.
What about the current situation to close this pull request? What can I do?
Thanks.

Comment (shciubrcom) [reply package-lock.json]: You can revert changes for this file or create new PR without it.

**Pull request merged**